go/chat/server.go

```diff
@@ -1471,6 +1442,25 @@ func (h *Server) GenerateOutboxID(ctx context.Context) (res chat1.OutboxID, err
 	return storage.NewOutboxID()
 }
 
+func (h *Server) runSlashCommands(ctx context.Context, uid gregor1.UID, convID chat1.ConversationID,
```

Comment:
why not make this as part of the cmdSource? just to keep the logic in a single place

Reply:
Sure.